.changeset/friendly-cats-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/rag-ai': patch
```

Comment:
I think this is a good change to bump this as major and go 1.0.0

Reply:
done.